includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
```

Comment:
If we define the text domain in this file, it means we need to make it consistent with all files that contain the static message. Even define a text domain in this file for this merge but the next coming merge, we need to refactor it again.

For this time I will remove all translation function from this file to make it simpler and prevent misleading.

I saw that article and pay attention to it as well. As can be seen from I have developed [a common function to be reused to translate the message](https://github.com/omise/omise-woocommerce/blob/feature/i18n/omise-util.php#L84).

At the beginning of this feature development, I already know that the translation will be developed or merged with another branch, feature/i18n, so I decided to focus on the scope of the feature. For the other stuffs, I still keep it intact like before I contribute with this project, if it not break, conflict, or affect the system to not working.

So one of benefits of this decision is to reduce the scope of review and testing time and cost.

However, thank you for your mention on the bad old code.